xlators/features/changelog/src/changelog-rpc-common.c

```diff
@@ -78,7 +78,8 @@ changelog_rpc_client_init(xlator_t *this, void *cbkdata, char *sockfile,
 dealloc_rpc_clnt:
     rpc_clnt_unref(rpc);
 dealloc_dict:
-    dict_unref(options);
+    if (options)
```

Comment:
In what scenario it is possible options is being NULL at this point?

Reply:
This was detected by coverity.